services/vehicle-assembly-process-defect-detection-model-service/app/core/exceptions.py

```
@@ -0,0 +1,675 @@
+    def clean_for_json(obj):
+        if isinstance(obj, bytes):
+            return f"<bytes object: {len(obj)} bytes>"
+        elif isinstance(obj, dict):
+            return {k: clean_for_json(v) for k, v in obj.items()}
+        elif isinstance(obj, list):
+            return [clean_for_json(item) for item in obj]
+        else:
+            return obj
+
+    # 에러 세부사항 정리
+    cleaned_errors = clean_for_json(exc.errors())
```

Ruff (0.12.2)

Local variable `cleaned_errors` is assigned to but never used

Remove assignment to unused variable `cleaned_errors`

(F841)